prod.compose.yaml

```diff
@@ -0,0 +1,30 @@
+version: "3.9"
+
+services:
+  postgres:
+    container_name: "snailycad-bot-postgres"
+    image: postgres
+    command: -p ${DB_PORT:-55545}
+    env_file:
+      - .env
+    # ports:
+    #   - "${DB_PORT:-55545}:${DB_PORT:-55545}"
```

Comment:
```suggestion
    ports:
      - "${DB_PORT:-55545}:${DB_PORT:-55545}"
```

Reply:
If we make ports visible to outside world, is a big security issue if the user is not config firewall correctly.